README.md

```diff
@@ -1 +1,21 @@
-Convert characters to their emoji counterparts.
\ No newline at end of file
+Dear users attentive to details enough to read this file,
+
+This is NOT a software to convert a word into an emoji of similar meaning. 
+
+What this IS is a software to convert characters to their emoji counterparts character by character, i.e. stylising your text into emojis (like what you will do when choosing a font). For example, "abcdef" converts to "🇦 🇧 🇨 🇩 🇪 🇫". The spaces are necessary as the english characters are actually Regional Indicator Symbol Letter emojis which would turn into flags of some countries if placed together without the spaces.
```

Comment:
> The spaces are necessary as the english characters are actually Regional Indicator Symbol Letter emojis which would turn into flags of some countries if placed together without the spaces.

The spaces are necessary as the **English** characters are ~actually~ [Regional Indicator Symbol Letter](https://en.wikipedia.org/wiki/Regional_indicator_symbol#:~:text=The%20regional%20indicator%20symbols%20are,that%20allows%20optional%20special%20treatment.) emojis which would turn into flags of some countries if placed together without the spaces.

List of changes:

- [ ] Include a hyperlink
- [ ] English instead of english
- [ ] Remove "Actually"